evalml/pipelines/binary_classification_pipeline.py

```diff
@@ -52,7 +52,7 @@ def _predict(self, X, objective=None):
         predictions = self._predict_with_objective(X, ypred_proba, objective)
         return infer_feature_types(predictions)
 
-    def predict_proba(self, X):
+    def predict_proba(self, X, X_train=None, y_train=None):
```

Comment:
Are you talking about https://github.com/alteryx/evalml/pull/2649 ? I think we want to make the api for predict/predict_proba the same for all classes that inherit from PipelineBase so that AutoMLSearch and other modules that use pipelines don't have to check if the pipelines are time series pipelines or not befor trying to pass in `X_train` and `y_train`.

Reply:
Ah got it, thanks for explaining!